integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktApplication.java

```diff
@@ -45,15 +50,28 @@ public static void main(String[] args) {
                 return;
             }
 
-            new SpringApplicationBuilder(IntegrasjonspunktApplication.class)
+            ConfigurableApplicationContext context = new SpringApplicationBuilder(IntegrasjonspunktApplication.class)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "context" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "context". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)